src/vmspawn/vmspawn.c

```diff
@@ -90,6 +92,7 @@ static int help(void) {
                "     --qemu-vsock=BOOL      Configure whether to use qemu with a vsock or not\n"
                "     --vsock-cid=           Specify the CID to use for the qemu guest's vsock\n"
                "     --tpm=BOOL             Configure whether to use a virtual TPM or not\n"
+               "     --kernel=PATH          Specify the kernel for direct kernel boot\n"
```

Comment:
Maybe we should name this --linux= instead of --kernel to more clearly indicate we're talking about a linux kernel image here.